src/styles/_card.scss

```diff
@@ -0,0 +1,123 @@
+@use './variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: variables.$width-card;
```

Comment:
Ensure that the card width is set to a fixed value of 200px as specified in the task requirements. The current value should be checked to ensure it matches the requirement.